src/main/resources/peer-review-transform.xsl

```diff
@@ -150,12 +160,15 @@
   </xsl:template>
   
   <xsl:template match="supplementary-material">
-    <dd class="supplementary-material">
-      <a class="supplementary-material__label coloration-white-on-color" href="#">
-        <xsl:value-of select="label" />
-      </a>
+      <dd class="supplementary-material">
+        <a class="supplementary-material__label coloration-white-on-color" 
+           href="{concat('file?id=10.1371/journal.',@id,'&amp;type=supplementary')}" 
+           title="{concat('Download ',plos:get-file-extension(normalize-space(caption/p/named-content)),' file')}"
```

Comment:
This looks like a more appropriate fn for this case: http://www.xsltfunctions.com/xsl/fn_string-join.html

Reply:
ok. let me read up on function, and then update test. thank you.